src/utils/efd.h

```diff
@@ -1,6 +1,7 @@
 /*
     Copyright (c) 2012-2013 Martin Sustrik  All rights reserved.
     Copyright 2015 Garrett D'Amore <garrett@damore.org>
+    Copyright (c) 2015-2016 Jack R. Dunaway.  All rights reserved.
```

Comment:
All that said, the copyright notice is yours, so you can make it what you want (within limits).  The above message was mostly intended as an educational FYI.  :-)